perses/app/setup_relative_calculation.py

```diff
@@ -1024,5 +1013,30 @@ def _resume_run(setup_options):
         raise("Can't resume")
 
 
+def _generate_htf(phase: str, topology_proposal_dictionary: dict, setup_options: dict):
+    """
+    Generates topology proposal for phase.
+    """
+    factory_name = setup_options['hybrid_topology_factory']
+    if factory_name == HybridTopologyFactory.__name__:
+        factory = HybridTopologyFactory
+    elif factory_name == RESTCapableHybridTopologyFactory.__name__:
+        factory = RESTCapableHybridTopologyFactory
+    try:
+        htf = factory(topology_proposal_dictionary[f'{phase}_topology_proposal'],
+                      topology_proposal_dictionary[f'{phase}_old_positions'],
+                      topology_proposal_dictionary[f'{phase}_new_positions'],
+                      neglected_new_angle_terms=topology_proposal_dictionary[f"{phase}_forward_neglected_angles"],
+                      neglected_old_angle_terms=topology_proposal_dictionary[f"{phase}_reverse_neglected_angles"],
+                      softcore_LJ_v2=setup_options['softcore_v2'],
+                      interpolate_old_and_new_14s=setup_options['anneal_1,4s'],
+                      rmsd_restraint=setup_options['rmsd_restraint']
+                      )
```

Comment:
@ijpulidos : I think you are missing a bunch of `HybridTopologyFactory` and `RESTCapableHybridTopologyFactory` arguments: https://github.com/choderalab/perses/blob/main/perses/app/relative_point_mutation_setup.py#L444-L460.

Reply:
Most of the ones in the code I linked are for the `HybridTopologyFactory`. The only ones that are used for the `RESTCapableHybridTopologyFactory` are: `rest_radius=rest_radius, w_scale=w_scale` -- you'll want to make sure the yaml has options for specifying these.

Reply:
Yes, you are correct.